cita-executor/core/src/cita_executive.rs

```diff
@@ -123,50 +130,42 @@ impl<B: DB + 'static> CitaExecutive<B> {
         }
 
         let init_gas = t.gas - U256::from(base_gas_required);
-        let _result = match t.action {
+        let result = match t.action {
             Action::Store | Action::AbiStore => {
                 // Maybe use tx_gas_schedule.tx_data_non_zero_gas for each byte store, it is more reasonable.
                 // But for the data compatible, just let it as tx_gas_schedule.create_data_gas for now.
                 let store_gas_used = U256::from(t.data.len() * tx_gas_schedule.create_data_gas);
-                if let Some(gas_left) = init_gas.checked_sub(store_gas_used) {
-                    Ok(TxExecResult { gas_left })
-                } else {
-                    return Err(ExecutionError::VM(VMError::NotEnoughBaseGas));
-                }
+                let mut receipt = Receipt::default();
+                receipt.quota_used = store_gas_used.as_u64();
+                Ok(receipt)
```

Comment:
How to handle error situation, such as `init_gas < store_gas_used`?

Reply:
I need some logic to handle the result, but haven't finished yet, I plan to handle this situation there.
But maybe it is better to handle this situation here. I will change this logic.